ha/core/cluster/cluster_manager.py

```diff
@@ -293,3 +294,24 @@ def controller_list(self) -> list:
             [list]: list of controllers.
         """
         return list(self._controllers.keys())
+
+    def process_request(self, action, args, output):
+        """
+        Generic method to handle process request
+
+        Args:
+            action ([string]): Take cluster action for each request.
+            args ([dict]): Parameter pass to request to process.
+        """
+        self._output = output
+        # TODO Add validater
+        if action == const.CLUSTER_COMMAND:
```

Comment:
This code is temporary right ? can you please add comment stating the same here ?

Reply:
Done